src/Compilers/CSharp/Portable/Compilation/CSharpDiagnosticFilter.cs

```diff
@@ -119,7 +119,8 @@ internal static class CSharpDiagnosticFilter
         ///     1. Warning level
         ///     2. Syntax tree level
         ///     3. Compilation level
-        ///     4. Global warning level
+        ///     4. Global analyzer config
+        ///     5. Global warning level
```

Comment:
How is 'global warning level' different than 'warning level'? I'm trying to understand this list and keep coming up confused.

Reply:
We could probably make the wording clearer. Warning level is the `/warn:3` that enables/disables big swathes of warnings.

Global warning level is `/warnaserror:+/-` that bumps all warnings up to errors

---
In reply to: [473388737](https://github.com/dotnet/roslyn/pull/45871#discussion_r473388737) [](ancestors = 473388737)